pkg/Library/CoreXenon/Designer/Structure.js

```diff
@@ -41,6 +44,22 @@ const createOrReplaceObject = async (layer, objectId, node, type, state, layout)
   return objectChanged(layer, id);
 };
 
+export const createObject = async (layer, {key, value}) => {
```

Comment:
thank you! a separate `createObject` method is so much better then reusing `morph`...

nit - the `createOrReplaceObject` method above can be just `replaceObject` now